### DISCUSSION TITLE
It's showing my account has been flagged and I am not able to connect to organisation repo

### DISCUSSION THREAD

**Discussion by nitesh-kinesiis** [community/community#80793] [New to GitHub]
![IMG_20231218_170544](https://github.com/community/community/assets/154225734/cc7a69d7-4ac9-41e6-a758-db9055a27372)

**Comment by queenofcorgis** [comment]
Hi @nitesh-kinesiis ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
